bot.py

```diff
@@ -3,13 +3,23 @@
 """This module deploys discord bot using discord.py library."""
 
 import time
+import datetime
 import os
+from typing import List, Optional
+from contextlib import suppress
 import discord
 from discord import app_commands
-from discord.ext import commands
+from discord.ext import commands, tasks
 from dotenv import load_dotenv
 
 load_dotenv()  # loads your local .env file with the discord token
+DISCORD_TOKEN: Optional[str] = os.getenv("DISCORD_TOKEN")
+
+# pylint: disable=invalid-name
+embed_channel_id = int()
+embed_message_id = int()
+embed_description = str()
+# pylint: enable=invalid-name
```

Comment:
Czy dobrze rozumiem, że jeśli wyłączymy i włączymy bota to utraci on wiedzę o tym jak znaleźć swoją ostatnią wiadomość?

Reply:
Tak, bot nie korzysta w tym momencie z zewnątrznej bazy danych.

Reply:
Czy chcesz, żeby było coś w rodzaju zapisu do pliku tekstowego w folderze bota? Mogę również dodać coś w rodzaju formularza, gdzie w przypadku zresetowania bota, można wkleić i  przypomnieć ID ostatniej wiadomości. W przypadku drugiej opcji pojawia się niestety problem potrzeby manualnego przypominania oraz brak możliwości odświeżania wiadomości, przed jej przypomnieniem.

Reply:
Zrobiłem tak jak zasugerowałeś.